src/sushi.sol

```diff
@@ -129,29 +137,29 @@ contract SushiJoin is CropJoin {
 
     function crop() internal override returns (uint256) {
         if (live) {
-            // withdraw of 0 will give us only the rewards
-            masterchef.withdraw(pid, 0);
+            // This can possibly fail if no rewards are owed, but that doesn't really matter as we wouldn't get any rewards anyways
+            try masterchef.harvest(pid, address(this)) {} catch {}
```

Comment:
We should have a test of this failure scenario so that we can ensure the passthrough works correctly here.

Reply:
Sure, I'll see if I can come up with something more explicit. This error actually happens quite frequently in the tests, but due to this being an integration test it depends on the state of the MasterChef.